html/pfappserver/root/src/views/Status/dashboard/_config/authentication.js

```diff
@@ -0,0 +1,46 @@
+import store from '@/store'
+import { modes, libraries } from '../_components/Chart'
+
+export default [
+  {
+    name: 'Authentication', // i18n defer
+    groups: [
+      {
+        name: 'Authentication Sources', // i18n defer, requires sources
```

Comment:
Please remove everything after `i18n defer` otherwise the extraction script won't pick up the localizable string.